learn/getting_started/installation.md

```diff
@@ -1,8 +1,10 @@
-# Installation
+# Installation and supported operating systems
```

Comment:
Is it a problem that the installation guide's sidebar link is super long now?

```suggestion
# Installation and OS support
```

This is shorter, but does it make sense?

Reply:
OS is a well-known acronym, but I think we should use the full form in the title.